### PR TITLE
MULTIARCH-4001: ABI CI workflow for IBMZ (s390x) with SNO configuration

### PR DESCRIPTION
**Pull request by phani2898**
Added CI workflow for Agent Based Installation (ABI) on IBMZ(s390x) architecture using ibmcloud with SNO configuration

This workflow will do the following : 
1. Creates an SNO cluster on ibmcloud using zVSI machines 
2. Verifies if all the SNO cluster operators are ready
3. Deletes the SNO cluster and underlying services that are created

### CODE REVIEW

**Comment by jaypoulz** [comment]
@phani2898 In order for us to see a rehearsal run, you'll need to add a job that invokes it.

Take a look like the ci-operator/config/openshift/multiarch tree for examples. Once you've added the job, you can run `$ make jobs` to update the prow configs.

**Comment by jaypoulz** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by jaypoulz** [comment]
Are we sure we want to be running this job as a presubmit. I have a few concerns:
1. Pre-merge payloads don't support s390x, so the best we can do is check a recent nightly (which won't have the installer code change it's supposed to be testing).
2. If we add a pre-merge job as required, one will be spawned with every PR. Do we have enough leases/capacity/funding to handle that?

--

My recommendation is to add a periodic job instead. :) Something like: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml#L132-L155

You can work with @tvardema to figure out a sane chron timing.

**Comment by phani2898** [comment]
> Are we sure we want to be running this job as a presubmit. I have a few concerns:
> 
> 1. Pre-merge payloads don't support s390x, so the best we can do is check a recent nightly (which won't have the installer code change it's supposed to be testing).
> 2. If we add a pre-merge job as required, one will be spawned with every PR. Do we have enough leases/capacity/funding to handle that?
> 
> --
> 
> My recommendation is to add a periodic job instead. :) Something like: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml#L132-L155
> 
> You can work with @tvardema to figure out a sane chron timing.

@jaypoulz , Sure My idea was to add as a periodic job but no periodic workflow is configured for agent-installer-utils so went with existing pre-submit.

Will work with @tvardema to add this

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
> @tvardema: job(s): pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno either don't exist or were not found to be affected, and cannot be rehearsed

This is because, job has been migrated from pre-submits to periodics

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (andfasano) [inline ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml:40]: Not sure what's the recommended way to handle ci-operator configs for arch different than x86, but in any case I guess that the openshift/installer folder may be a better target rather than agent-installer-utils

Comment (phani2898) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml:40]: @andfasano 

In consultation with @deepsm007 from CI we have added this config for s390x to target latest builds for the same in our workflow. 

As per our previous discussion in January along with @v78singh , we came to a conclusion that it's okay if we keep s390x workflows in a new directory under agent-intsaller-utils itself. Please confirm this.

Comment (andfasano) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml:40]: Steps registry folder to store ibmz workflow it's ok.
My concern it's about placing s390x _jobs_ inside `.../config/openshift/agent-installer-utils`, which is a minor one used by ABI - and moreover rarely updated. The main repo for ABI is `.../config/openshift/installer`, which seems to me a more suitable config repo for such jobs.

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (phani2898) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml]: Removed the s390x images from presubmits as we will be running periodics now.

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (andfasano) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml]: The comment was for all the types of jobs, not just the presubmits. The `agent-installer-utils` component repo folder it's not the right place for a ibmz periodic job testing the whole installation. As in general I'm not opposed in having periodics outside the [relase folder](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/#periodics-outside-of-openshiftrelease), there could be better alternatives for placing the periodic - for example `openshift/multiarch` seems a suitable config folder ?

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (phani2898) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml]: @andfasano , okay I believe there are other e2e workflows which does the e2e whole cluster installation on x86 in the same agent-installer-utils directory.

@tvardema , Please suggest where to place the e2e agent based installation on IBMZ workflow periodic job

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (andfasano) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml]: @phani2898 that's the common pattern to verify the current repo (in this case `openshift/agent-installer-utils`) patches, because for each PR (commit) the CI creates an ephemeral payload assembled with the latest available release version + a fresh component image (usually, to be promoted) built with the content of the current patch.. So, the e2e _presubmits  jobs_ are defined to verify any new specific `agent-installer-utils` before merging it.

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-utils-release-4.15-periodics-e2e-agent-ibmz-sno

File: ci-operator/step-registry/agent/ibmz/OWNERS
Comment (jaypoulz) [inline ci-operator/step-registry/agent/ibmz/OWNERS:2]: I'm fine with landing this as is, but can we have a follow up patch to add:
- multiarch-approvers
and
- multiarch-reviewers

to all of the added sections?

Comment (phani2898) [reply ci-operator/step-registry/agent/ibmz/OWNERS:2]: Sure @jaypoulz, Please let me know whom to add

File: ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml
Comment (phani2898) [reply ci-operator/config/openshift/agent-installer-utils/openshift-agent-installer-utils-release-4.15.yaml]: okay sure @andfasano , @jaypoulz / @tvardema Please suggest where to place the e2e agent based installation on IBMZ workflow periodic job.

File: ci-operator/step-registry/agent/ibmz/OWNERS
Comment (tvardema) [reply ci-operator/step-registry/agent/ibmz/OWNERS:2]: You would add literally what he typed above.  In the `approvers:` section, add `- multiarch-approvers`, and the same for the reviewers section.

Comment (phani2898) [reply ci-operator/step-registry/agent/ibmz/OWNERS:2]: Added the multiarch-approvers and reviewers in the OWNERS, please re-review :)

Comment (phani2898) [reply ci-operator/step-registry/agent/ibmz/OWNERS:2]: yeah @tvardema, I understood that later on when I was checking other OWNERS files, Earlier I thought some names will be pointed out 😅

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-agent-installer-release-4.15-periodics-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by tvardema** [comment]
I think the only things missing are some follow-ups to config bits above, a clean test run if possible, and a 4.16 job definition to continue testing on versions above 4.15.

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse pull-ci-openshift-agent-installer-utils-release-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Review by jaypoulz — APPROVED** [review]
/lgtm

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Comment by phani2898** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-e2e-agent-ibmz-sno periodic-ci-openshift-multiarch-master-nightly-4.16-e2e-agent-ibmz-sno

**Pull request merged**
